src/aldente_server.cpp

```diff
@@ -1,10 +1,68 @@
 #include "aldente_server.h"
 
+#include "util/config.h"
+#include "game/game_state.h"
+#include "physics.h"
+#include "scene_manager.h"
+#include "net/network_manager.h"
 #include "timer.h"
 #include <chrono>
 #include <iostream>
+#include <boost/filesystem.hpp>
+
+
+AldenteServer::~AldenteServer() {
+    GeometryGenerator::destroy();
+    glfwTerminate();
+}
+
+static void glSetup() {
```

Comment:
That makes sense.  Kavin said it'll take a little bit of refactoring, so I'd like to let @AznCelebration make those changes in a future PR.